.github/workflows/download_pipeline.yml

```
@@ -0,0 +1,67 @@
+name: Test successful pipeline download with 'nf-core download'
+
+# Run the workflow when:
+#  - dispatched manually
+#  - when a PR is opened or reopened to master branch
+#  - the head branch of the pull request is updated, i.e. if fixes for a release are pushed last minute to dev.
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - opened
+    branches:
+      - master
+  pull_request_target:
+    branches:
+      - master
+
+env:
+  NXF_ANSI_LOG: false
+
+jobs:
+  download:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Nextflow
+        uses: nf-core/setup-nextflow@v1
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+          architecture: "x64"
+      - uses: eWaterCycle/setup-singularity@v7
+        with:
+          singularity-version: 3.8.3
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install git+https://github.com/nf-core/tools.git@dev
+
+      - name: Get the repository name and current branch set as environment variable
+        run: |
+          echo "REPO_LOWERCASE=${GITHUB_REPOSITORY,,}" >> ${GITHUB_ENV}
+          echo "REPOTITLE_LOWERCASE=$(basename ${GITHUB_REPOSITORY,,})" >> ${GITHUB_ENV}
+          echo "REPO_BRANCH=${GITHUB_REF#refs/heads/}" >> ${GITHUB_ENV}
```

this is unfortunately broken in this version of the template
```suggestion
         
          echo "{% raw %}REPO_BRANCH=${{ github.event.inputs.testbranch || 'dev' }}" >> ${GITHUB_ENV}
```